client/state/sites/plans/selectors/get-plan-discounted-raw-price.ts

```diff
@@ -12,23 +12,27 @@ export function getPlanDiscountedRawPrice(
 	productSlug: string,
 	{
 		returnMonthly,
+		returnSmallestUnit,
 	}: {
 		/**
-		 * If true, attempt to calculate and return the monthly price. Note that this
-		 * is not precise as it relies on float division and could have rounding
-		 * errors.
+		 * If true, attempt to calculate and return the monthly price.
 		 */
 		returnMonthly?: boolean;
+		/**
+		 * If true, represent price as a currency's smallest unit ( as integers
+		 * instead of floats ). If precision matters, set to true, as otherwise the
+		 * price relies on float division and could have rounding errors.
+		 */
+		returnSmallestUnit?: boolean;
 	} = {}
 ) {
 	const plan = getSitePlan( state, siteId, productSlug );
+	const rawPrice = plan?.rawPrice ?? -1;
 
-	if ( ( plan?.rawPrice ?? -1 ) < 0 || ! isSitePlanDiscounted( state, siteId, productSlug ) ) {
+	if ( rawPrice < 0 || ! isSitePlanDiscounted( state, siteId, productSlug ) || ! plan ) {
```

Comment:
Nit: by rawPrice < 0, it actually imples `! plan` from how rawPrice is derived. We can also move `if ( ! plan ) { return null; }` right above `rawPrice` since we know the subsequent computation isn't needed if `plan` is falsey. It's very minor so please keep it for now :)